easybuild/easyconfigs/s/Siesta/Siesta-4.0-foss-2017b.eb

```diff
@@ -0,0 +1,18 @@
+name = 'Siesta'
+version = '4.0'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+vl = version.split('-')
```

Comment:
@jrbosch This is kind of pointless, there's no `-` in the version (anymore)?

So you can change this to:

```python
source_urls = ['https://launchpad.net/siesta/%(version_major_minor)s/%(version)s/+download']
```

The `version_major_minor` is not needed now, but will help with future updates, like for 4.0.1 which is available at https://launchpad.net/siesta/4.0/4.0.1/+download/siesta-4.0.1.tar.gz